.werft/util/certs.ts

```diff
@@ -33,7 +33,46 @@ export async function issueCertificate(werft: Werft, params: IssueCertificatePar
 
     exec(`echo "Domain: ${params.domain}, Subdomains: ${subdomains}"`, { slice: shellOpts.slice });
     validateSubdomains(werft, shellOpts.slice, params.domain, subdomains);
-    createCertificateResource(werft, shellOpts, params, subdomains);
+
+    const maxAttempts = 5
+    const timeout = "150s"
+    var i: number
+    var certReady: boolean
```

Comment:
Tiny nit: Would prefer to initialise and then not specify the type rather than specifying the type and not the initial value:

```ts
var i = 0
var certReady = false
```